devel/libtapi/Portfile

```diff
@@ -19,53 +20,32 @@ long_description        ${description} Newer MacOS SDKs contain only these TAPI
 
 homepage                http://opensource.apple.com/source/tapi/
 
-checksums               rmd160  a99aa279e626d4d638281ef9555d9b5db5f15145 \
-                        sha256  66c715baaea6f28e8c079f36189027a52744138c38e83e97aeb399232365138f \
-                        size    32205809
-
-# use cmake-bootstrap to minimize dependencies.
-depends_build-replace   path:bin/cmake:cmake port:cmake-bootstrap
-configure.cmd           ${prefix}/libexec/cmake-bootstrap/bin/cmake
-depends_skip_archcheck-append cmake-bootstrap
-
-platform darwin {
-    # pick a suitable python to build with
-    if {${os.major} < 11} {
-        # system python is too old, use python27-bootstrap to minimize dependencies
-        depends_build-append    port:python27-bootstrap
-        configure.args-append   -DPYTHON_EXECUTABLE=${prefix}/libexec/python27-bootstrap/bin/python2.7
-        depends_skip_archcheck-append python27-bootstrap
-    } elseif {${os.major} > 19} {
-        # Use system python 3.x
-        configure.args-append   -DPYTHON_EXECUTABLE=/usr/bin/python3
-    } else {
-        # use the system python, which is known to be fine for darwin 11 to 19
-        configure.args-append   -DPYTHON_EXECUTABLE=/usr/bin/python2.7
-    }
-}
+checksums               rmd160  d1ce5001e4ec0452c9a6e2464d72308d0058fdf1 \
+                        sha256  90be4f254bf5df5fc22853536791f533db953c0f772a11f7ea66601fce21d003 \
+                        size    38640440
+
+set python_branch       3.11
+set python_version      [string map {. {}} ${python_branch}]
+
+depends_build-append    port:python${python_version}
+configure.args-append   -DPYTHON_EXECUTABLE=/usr/bin/python${python_branch}
```

Comment:
@catap Should be `${prefix}/bin/python${python_branch}`, I believe.

Reply:
My point was that `/usr/bin/` is wrong.

Reply:
Well... it works on 10.5 that means that python doesn't really required. Let me double check it and maybe I may rescue support of old cmake

Reply:
@Gcenx an attempt to use old cmake (from cmake-bootstrap) leads to failing due missed `FindPython3`. Because it is build => no python is required.

Right now I'm retesting it on 10.5 without any python, let see how it goes.

Reply:
Configure somehow picks the correct `python311` in Macports prefix. But the config arg should still be fixed.

Reply:
Nope, it fails due to missed python:
```
:info:configure -- Looking for os_signpost_interval_begin
:info:configure -- Looking for os_signpost_interval_begin - not found
:info:configure CMake Error at /opt/local/share/cmake-3.24/Modules/FindPackageHandleStandardArgs.cmake:230 (message):
:info:configure   Could NOT find Python3 (missing: Python3_EXECUTABLE Interpreter)
:info:configure       Reason given by package: 
:info:configure           Interpreter: Wrong major version for the interpreter "/usr/bin/python"
:info:configure Call Stack (most recent call first):
:info:configure   /opt/local/share/cmake-3.24/Modules/FindPackageHandleStandardArgs.cmake:594 (_FPHSA_FAILURE_MESSAGE)
:info:configure   /opt/local/share/cmake-3.24/Modules/FindPython/Support.cmake:3203 (find_package_handle_standard_args)
:info:configure   /opt/local/share/cmake-3.24/Modules/FindPython3.cmake:490 (include)
:info:configure   CMakeLists.txt:721 (find_package)
:info:configure -- Configuring incomplete, errors occurred!
```

=> so it picks it from a path on my test build. good.

@barracuda156 thanks for notice the wrong path.